app/src/androidTest/java/com/github/se/project/ui/profile/CreateTutorProfileTest.kt

```diff
@@ -178,4 +162,22 @@ class CreateTutorProfileTest {
         .onNodeWithTag("noProfile")
         .assertTextEquals("No Profile selected. Should not happen.")
   }
+
+  @Test
+  fun descriptionField_updatesCorrectly() {
+    (mockViewModel.currentProfile as MutableStateFlow).value = getMockProfile()
+
+    composeTestRule.setContent {
+      CreateTutorProfile(navigationActions = mockNavigationActions, mockViewModel)
+    }
+
+    composeTestRule
+        .onNodeWithTag("experienceField")
+        .assertTextContains("Do you have any experience as a tutor ?")
+
+    val descriptionText = "I have experience teaching math and physics."
+    composeTestRule.onNodeWithTag("experienceField").performTextInput(descriptionText)
+
+    composeTestRule.onNodeWithTag("experienceField").assertTextContains(descriptionText)
+  }
 }
```

Comment:
Great work! The test effectively checks if the description field updates correctly and handles the state well.